pkg/cli/schemaherokubectlcli/root.go

```diff
@@ -40,6 +40,7 @@ func RootCmd() *cobra.Command {
 	cmd.AddCommand(DescribeCmd())
 	cmd.AddCommand(UpdateCmd())
 	cmd.AddCommand(ApproveCmd())
+	cmd.AddCommand(GenerateCmd())
 
 	viper.BindPFlags(cmd.Flags())
```

Comment:
i know this is not part of this pr but this should probably be removed since its also being bound here https://github.com/schemahero/schemahero/pull/210/files#diff-342226c707192eb3a2c6977f59652bb6R23